public/js/clients/chrome.js

```diff
@@ -39,10 +41,15 @@ function createTabs(tabs) {
 
 function connectClient() {
   const deferred = defer();
-  fetch("/chrome-tabs").then(res => {
-    res.json().then((body) => {
-      deferred.resolve(createTabs(body));
-    });
+
+  if(!isEnabled("chrome.debug")) {
+    return deferred.resolve(createTabs([]))
+  }
```

Comment:
hmm - i'm not sure, i think it might be nice to flag firefox off at somepoint too